app/policies/namespace_policy.rb

```diff
@@ -42,10 +42,15 @@ def index?
 
   def create?
     raise Pundit::NotAuthorizedError, "must be logged in" unless user
-    (APP_CONFIG.enabled?("user_permission.manage_namespace") || user.admin?) && push?
+    (APP_CONFIG.enabled?("user_permission.create_namespace") || user.admin?) && push?
+  end
+
+  def update?
+    raise Pundit::NotAuthorizedError, "must be logged in" unless user
+    (user.admin? || (APP_CONFIG.enabled?("user_permission.manage_namespace") &&
+                     namespace.team.owners.exists?(user.id))) && push?
```

Comment:
I don't understand what you mean... I see no `owner?` method, and 
`namespace.team.owners.exists?(user.id)` is used everywhere else in this file...